apps/cms/priv/cms/schedule-pdfs.json

```diff
@@ -0,0 +1,118 @@
+[
```

Comment:
We have an updated API payload for this. You can view it here: https://test-mbta.pantheonsite.io/cms/schedules/350?_format=json